### DISCUSSION TITLE
使用dig命令挖掘方块失败

### DISCUSSION THREAD

**Discussion by hequano** [ORelio/Minecraft-Console-Client#2408] [Q&A] [answered]
使用dig命令时人物像是在挥手，并没有进行挖掘方块

**Comment by BruceChenQAQ** [comment]
The mining function is not perfect yet, MCC currently needs server side to calculate the mining status, but some plugins or modified servers may not handle it properly.

**Comment by BruceChenQAQ** [comment, marked as answer]
Try using the latest version in the pre-release and then check if the problem still exists.

If you still can't dig, you can try to specify the digging time manually.
Check [Breaking time](https://minecraft.fandom.com/wiki/Breaking#Blocks_by_hardness) Or [挖掘用时](https://minecraft.fandom.com/zh/wiki/%E6%8C%96%E6%8E%98#%E6%96%B9%E5%9D%97%E7%A1%AC%E5%BA%A6)

```
/dig <x> <y> <z>: Attempt to break a block

All Usages:
/dig
/dig <Duration(Unit: seconds, can contain decimals)>
/dig <Location>
/dig <Location> <Duration(Unit: seconds, can contain decimals)>
/dig _help
/dig _help -> /help dig
```
